### PR TITLE
Release 0.1

### PR DESCRIPTION
**Pull request by idovgalyov-4xxi**
Console-based game version with limited gameplay.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on format provided by [Keep a Changelog](https://keepachangelog.com) website.
+
+## [Unreleased]
+
```
Comment (D-Mantrid) [inline CHANGELOG.md:7]: Add some more information about upcoming features for the next release.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+
+For tests run:
+```bash
+python3 -m unittest
+```
+
+And for running test game session you can type:
+```bash
+python3 pyndemic.py test/test_input.txt 42
+```
```
Comment (D-Mantrid) [inline README.md:37]: I think we need to add a section with a possible command list after all for completeness.

File: CHANGELOG.md (hunk 1 above)
Comment (D-Mantrid) [inline CHANGELOG.md:7]: Add some more information about upcoming features for the next release.

File: README.md (hunk 2 above)
Comment (idovgalyov-4xxi) [reply README.md:52]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (idovgalyov-4xxi) [reply CHANGELOG.md:12]: Added.

**Pull request merged**